src/index.html

```diff
@@ -13,6 +13,38 @@
     />
```

Comment:
The stylesheet link should refer to a CSS file, not an SCSS file. SCSS files are preprocessed into CSS, so the browser can only understand the final CSS output. Make sure to link to the compiled CSS file instead.